ci/scripts/audit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/log.go
+  make audit
```

Comment:
`Makefile` needs updating with `audit` and `build` targets

Reply:
Pushed changes thanks :-)